controller/lib/stl/new.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#ifndef NEW_H
+#define NEW_H
+
+#include <stddef.h>
+
+#ifdef ARDUINO_AVR_UNO
```

Comment:
Should this be guarded on TEST_MODE instead?

Reply:
No; the STM32 has an STL, it turns out.  Which is fantastic news.  Should have written a comment.